packages/nc-gui/components/smartsheet/grid/canvas/utils/groupby.ts

```
@@ -206,6 +206,10 @@ export function generateGroupPath(data?: CanvasGroup) {
   return path
 }
 
+export function comparePath(pathA?: Array<number | string> | null, pathB?: Array<number | string> | null) {
+  return (ncIsArray(pathA) ? pathA : []).join() === (ncIsArray(pathA) ? pathA : []).join()
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)